src/components/Payment/Payment.tsx

```diff
@@ -272,7 +273,8 @@ const Payment = ({
       )}
       <div className={panelClassName}>
         <div className={panelHeaderClassName}>
-          <h3>{t('user:payment.payment_method')}</h3>
+          {!isChangingOffer && <h1 className="hideUntilFocus">{t('nav.payments')}</h1>}
```

Comment:
Isn't this the second h1?

Reply:
See https://github.com/Videodock/ott-web-app/pull/3#discussion_r1434917952